packages/terra-arrange/src/Arrange.scss

```diff
@@ -10,18 +10,16 @@
   }
 
   // Makes sure fill expands to fill the remaining space of its parent container
-  // Display set to ensure correct appreance if applied to inline elements like span
   .fill {
-    display: block;
-    flex: 1;
-    flex-basis: auto;
+    flex: 1 1 auto;
     min-width: 0;
   }
 
   // Styles specific to align items independently of each other
   .fit {
     align-self: flex-start;
-    display: flex;
+    flex: 0 0 auto;
+    overflow: auto;
```

Comment:
Is there a reason we are using overflow:auto here? Are we expecting content to scroll if it's beyond it's max width?

Reply:
I put it in here for when say you have content (say images) that is too wide in the start and end containers.  This would allow them to shrink down if given a max-width rather than push the end-content off the screen.  However, now since we are passing in the props for the fit containers, this isn't really needed anymore.